crates/containerd-shim-wasmedge/src/lib.rs

```diff
@@ -2,3 +2,30 @@ pub mod error;
 pub mod executor;
 pub mod instance;
 pub mod oci_utils;
+
+#[cfg(test)]
+mod test {
+    #[cfg(feature = "static")]
+    #[test]
+    fn check_static_linking() {
+        let output = std::process::Command::new("ldd")
```

Comment:
Can we assume that `ldd` is installed? Maybe skip the test if the binary is not present.

Reply:
We can avoid using `lld` by using `libc::dladdr`.
I've pushed a new version using that instead :-)